src/term-spec.js

```diff
@@ -32,7 +32,7 @@ declare export class NamedObjectProperty extends ObjectProperty {
   name: PropertyName;
 }
 declare export class MethodDefinition extends NamedObjectProperty {
-  body: FunctionBody;
+  body?: FunctionBody; // undefined only on abstract class methods
```

Comment:
This is not ideal. There are certainly other options but I'm not sure which one is best, so I went with this "minimum impact" approach for now.